schemas/test/en/test_list_collector_list_summary.json

```diff
@@ -0,0 +1,497 @@
+{
```

Comment:
This should be `test_list_collector_section_summary`.
The existing one on master called `test_list_collector_section_summary` should be `test_list_collector_list_summary`